### DISCUSSION TITLE
Support raw image files

### DISCUSSION THREAD

**Discussion by Ftbom** [Difegue/LANraragi#629] [Ideas]
Please support manga files that be structured below.

```
├─── Name
|    |    ├─── Page 1.png
|    |    ├─── Page 2.png
|    |    └─── Page 3.png
|    └─── ...
```

I store my mangas in googledrive and it is really slow to load the archive files.So I decide to store the mangas in raw images which LANraragi has not supported yet.

**Comment by nayumiDEV** [comment]
Why do you store it on Google Drive???

**Comment by Difegue** [comment]
Unlikely to be done on my part - Most of the logic is based around (mostly) immutable IDs for single archive files and adding sparse folders would be incredibly messy.  

Moving to a discussion 🙃
